tests/k8s/k8s-gsg-test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+NAMESPACE="kube-system" 
+
+source "../helpers.bash"
+source /home/vagrant/.profile
+function cleanup {
+	kubectl delete -f https://raw.githubusercontent.com/cilium/cilium/master/examples/minikube/l3_l4_l7_policy.yaml
+	kubectl delete -f https://raw.githubusercontent.com/cilium/cilium/master/examples/minikube/l3_l4_policy.yaml
+	kubectl delete -f https://raw.githubusercontent.com/cilium/cilium/master/examples/minikube/demo.yaml
+	kubectl delete -f cilium-ds.yaml
+	kubectl delete -f https://raw.githubusercontent.com/cilium/cilium/master/examples/kubernetes/rbac.yaml
+}
+
+trap cleanup exit
+
+echo "KUBECONFIG: $KUBECONFIG"
+
+cleanup
+
+until [ "$(kubectl get cs | grep -v "STATUS" | grep -c "Healthy")" -eq "3" ]; do 
+	echo "---- Waiting for cluster to get into a good state ----"
+	sleep 5
+done
+
+
+echo "----- adding RBAC for Cilium -----"
+kubectl create -f https://raw.githubusercontent.com/cilium/cilium/master/examples/kubernetes/rbac.yaml
+
+echo "----- deploying Cilium Daemon Set onto cluster -----"
+wget https://raw.githubusercontent.com/cilium/cilium/master/examples/kubernetes/cilium-ds.yaml
+sed -i s/"\/var\/lib\/kubelet\/kubeconfig"/"\/etc\/kubernetes\/kubelet.conf"/g cilium-ds.yaml
```

Comment:
small nit, the symbol in sed can be replaced with other one, for example `echo foo | sed s+foo+bar+g` returns "bar". That way you don't need to escape `/`